yaksh/test_models.py

```diff
@@ -143,24 +149,33 @@ def test_dump_questions_into_json(self):
         """ Test dump questions into json """
         question = Question()
         question_id = [self.question2.id]
-        questions = json.loads(question.dump_into_json(question_id, self.user2))
+        questions_zip = question.dump_questions(question_id, self.user2)
+        zip_file = zipfile.ZipFile(questions_zip, "r")
+        zip_file.extractall("/tmp/")
```

Comment:
I had asked Aditya to make this change, though my intention was the same - the file should be extracted/created in a temp directory and then cleared out. I should have made this more clear.